### PR TITLE
Hide user settings in Demo Mode

### PR DESCRIPTION
**Pull request by meisenzahl**
Closes https://github.com/elementary/wingpanel-indicator-session/issues/63

### CODE REVIEW

**Comment by zeebok** [comment]
Is this still an issue with eOS7? I am fairly sure in the early access build that you create a user when you live boot so this wouldn't be needed anymore. But maybe that is changing for release? Hmm I guess that is a question for @danirabbit to answer. If this is something that will be an issue in the future this is definitely a good patch!

**Comment by meisenzahl** [comment]
I tested this with a daily build ISO of OS 7. Screenshots are taken in Demo Mode.

File: src/Utils.vala [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program. If not, see http://www.gnu.org/licenses/.
+ */
+
+namespace Utils {
+    public static bool is_running_in_demo_mode () {
+        var proc_cmdline = File.new_for_path ("/proc/cmdline");
```
Comment (danirabbit) [inline src/Utils.vala:19]: Since this is only used in `indicator.vala`, maybe a private method there instead of creating a new utility class?

**Review by danirabbit — CHANGES_REQUESTED** [review]

**Review by danirabbit — APPROVED** [review]
Nice work! :rocket:

**Pull request merged**